src/rgw/rgw_aws_lambda.cc

```diff
@@ -0,0 +1,116 @@
+//
+// Created by prateek on 30/03/20.
+//
+
+#include <string>
+#include <iostream>
+#include <aws/core/Aws.h>
+#include <aws/core/utils/Outcome.h>
+#include <aws/lambda/LambdaClient.h>
+#include <aws/lambda/model/InvokeRequest.h>
+
+namespace lambda {
+    const char *ALLOCATION_TAG = "TEST";
```

Comment:
what is allocation tag?

Reply:
All the memory allocation functions in aws like `Aws::MakeShared`, require ALLOCATION_TAG for memory tracking.
Not sure how they use it.